### DISCUSSION TITLE
Channel list has a combination of other channels?

### DISCUSSION THREAD

**Discussion by andygodber** [jertel/vuegraf#51] [Q&A] [answered]
Hi Jason - I bought a Vue just so I could use your app, or was it the other way 'round ;)
Anyway - Im mostly up and running but have a question:
1. On initial discovery of the channels,  I get a device_name  xxxx-1,2,3 as well as xxx-1. is this the Vue's "Total Usage" measurement, and if so, why only 1-3?

 
![emp1](https://user-images.githubusercontent.com/1061812/120856513-e9400880-c577-11eb-87b2-8c60133e5027.PNG)

![emp2](https://user-images.githubusercontent.com/1061812/120856570-f78e2480-c577-11eb-8546-f92e96d89891.PNG)

**Comment by jertel** [comment, marked as answer]
Basement-1,2,3 represents the big clamp sensor, i.e., your panel feed. I have no idea why they put the 1,2,3 suffix on there but it's like that for everyone.

**Reply by magico13** [reply]
1,2,3 is theoretically for the 3 main clamps, added together as one channel. You only need 3 clamps with 3 phase power (industrial usage mostly) so really only 1 and 2 are used. At least that's my working theory on why it's labeled that way in the api
